### DISCUSSION TITLE
slave节点没有同步master

### DISCUSSION THREAD

**Discussion by zuoHx580** [apache/rocketmq#6621] [Q&A]
![image](https://user-images.githubusercontent.com/45234194/233235012-6e7584d1-1b1f-4789-b599-a47c3b1e0f51.png)

&nbsp;

应该是slave没有和master对齐，如何手动对齐呢

**Comment by RongtongJin** [comment]
> ![image](https://user-images.githubusercontent.com/45234194/233235012-6e7584d1-1b1f-4789-b599-a47c3b1e0f51.png)
> 
>  
> 
> 应该是slave没有和master对齐，如何手动对齐呢

看一下master和slave的store.log，日志应该有相关提示

**Reply by zuoHx580** [reply]
> > ![image](https://user-images.githubusercontent.com/45234194/233235012-6e7584d1-1b1f-4789-b599-a47c3b1e0f51.png)
> >  
> > 应该是slave没有和master对齐，如何手动对齐呢
> 
> 看一下master和slave的store.log，日志应该有相关提示

![image](https://user-images.githubusercontent.com/45234194/233235946-9af14e8d-1024-404e-aed1-b51fa09689db.png)

同步异常

**Reply by RongtongJin** [reply]
主的store.log有信息吗？另外用的版本是？

**Reply by zuoHx580** [reply]
> 主的store.log有信息吗？另外用的版本是？

这是主的store.log，版本为5.1.0
![image](https://user-images.githubusercontent.com/45234194/233236543-612b5d94-a84a-4454-8d37-e8fd7b7d3ee5.png)

**Reply by RongtongJin** [reply]
似乎epochCheckpoint中的数据与commitLog中的数据无法对齐，是否有手动删除过相关文件呢？

建议下线清理epochFileCheckpoint和epochFileCheckpoint.bak后重启

**Reply by zuoHx580** [reply]
> 似乎epochCheckpoint中的数据与commitLog中的数据无法对齐，是否有手动删除过相关文件呢？
> 
> 建议下线清理epochFileCheckpoint和epochFileCheckpoint.bak后重启

有删过，就是因为没法同步，所以删过一次

刚刚又试了一遍，还是不行，slave的机子也没有生成epochFileCheckpoint

然后我把slave  ~/store下的文件全删了，再重启，就可以同步了
